llvm/include/llvm/Frontend/OpenMP/OMP.h

```diff
@@ -30,6 +31,14 @@ Directive getCompoundConstruct(ArrayRef<Directive> Parts);
 bool isLeafConstruct(Directive D);
 bool isCompositeConstruct(Directive D);
 bool isCombinedConstruct(Directive D);
+
+/// Create a nicer version of a function name for humans to look at.
+std::string prettityFunctionName(StringRef FunctionName);
```

Comment:
prettity?

Reply:
Only look at the last commit for my PRs. I fixed this and its 'prettify' in upstream now.